Schema/V1/no.ks.fiks.politisk.behandling.v1.einnsyn.utvalgssaker.send.schema.json

```diff
@@ -1,40 +1,16 @@
 {
   "$schema": "https://json-schema.org/draft/2020-12/schema",
-  "$id": "https://no.ks.fiks.politisk.behandling.v1.einnsyn/utvalgssaker.send.schema.json",
+  "$id": "no.ks.fiks.politisk.behandling.v1.einnsyn.utvalgssaker.send.schema.json",
   "title": "SendUtvalgssakerEinnsyn",
   "definitions": {
     "Utvalgssakeinnsyn": {
       "type": "object",
       "properties": {
         "referanseEksternNoekkel": {
-          "type": "object",
-          "properties": {
-            "fagsystem": {
-              "type": "string"
-            },
-            "noekkel": {
-              "type": "string"
-            }
-          },
-          "required": [
-            "fagsystem",
-            "noekkel"
-          ]
+          "$ref": "no.ks.fiks.politisk.behandling.v1.felles.eksternnoekkel.schema.json"
         },
         "utvalgetsSaksnummer": {
```

Comment:
Dette er vel fagsystemets saksnummer. Feltene for saksnummer og eksternnøkkel bør være konsistente over tjenestene. Utvalgets saksnummer er returdata (del av behandlingsplanen) og bør ha egen type.

Reply:
Hmmm. Ja, men betyr det noe mtp at vi bare sier at det er et "saksnummer"? Når vi genererer kode basert på skjema så betyr det at man putter inn et objekt av typen saksnummer, som er like selv om de er for fagsystem eller utvalg?